Essentials/src/messages_cs.properties

```diff
@@ -465,6 +465,8 @@ seenOffline=\u00a76Hrac \u00a7c{0} \u00a76je \u00a74offline\u00a76 od \u00a7c{1}
 seenOnline=\u00a76Hrac \u00a7c{0} \u00a76je \u00a7aonline\u00a76 od \u00a7c{1}\u00a76.
 sellBulkPermission=\u00a76Nemas opravneni hromadne prodavat.
 sellHandPermission=\u00a76Nemas opravneni manualne prodavat.
+cannotSellNamedItem=\u00a76You are not allowed to sell named items.
+cannotSellTheseNamedItems=\u00a76You are not allowed to sell these named items: \u00a74{0}
```

Comment:
PRs should only add messages to `messages.properties`, not the translated files - these will be filled in by Crowdin.